docs/src/features.md

```diff
@@ -127,31 +90,3 @@ You can change the default keybinding by remapping the command as below:
     ]
 },
 ```
-Note that <kbd>F12</kbd> may conflict with your Goto Definition keybinding. To avoid the conflict, make sure that you
-have a context which checks that the AC widget is not visible:
-```js
-{
-    "command": "lsp_symbol_definition",
-    "keys": [
-        "f12"
-    ],
-    "context": [
-        {
-            "key": "lsp.session_with_capability",
-            "operator": "equal",
-            "operand": "definitionProvider"
-        },
-        {
-            "key": "auto_complete_visible",
-            "operator": "equal",
-            "operand": false
-        }
-    ]
-},
-```
-There is an example of this in LSP's default keybindings.
```

Comment:
This part was in a very weird place, talking about F12 potentially conflicting with default binding after the example that shown how to override default F12 key binding without conflicting with "goto definition". I've deemed that part completely unnecessary and it was removed.